### PR TITLE
Fix gene and gene set queries

### PR DESCRIPTION
**Pull request by sandertan**
- Fix inconsistent use of geneSet and geneset
- Fix gene query from study summary
- Fix gene set query with gene profile selected:

screen shot 2017-05-11 at 19 29 08

### CODE REVIEW

File: portal/src/main/webapp/WEB-INF/jsp/visualize.jsp [hunk 1]
```diff
@@ -78,7 +78,7 @@
         Set<String> disabledTabs = new HashSet<String>(GlobalProperties.getDisabledTabs());
         List<String> disabledTabsForGenesets = Arrays.asList("IGV", "mutual_exclusivity", "enrichments", "survival", "network", "download", "bookmark", "mutations");
         //If we have a query with only gene sets (and no genes), we only want three tabs displayed: Oncoprint, Plots and Co-Expression
-        Boolean genesetQuery = localGeneSetList.length() > 0 && localGeneList.length() == 0;
+        Boolean genesetQuery = localGeneSetList != null && localGeneSetList.length() > 0 && localGeneList.length() == 0;
```
Comment (pieterlukasse) [inline portal/src/main/webapp/WEB-INF/jsp/visualize.jsp]: please change to `localGeneSetList`

Comment (sandertan) [inline portal/src/main/webapp/WEB-INF/jsp/visualize.jsp]: You probably meant: change to `localGenesetList`

Comment (sandertan) [inline portal/src/main/webapp/WEB-INF/jsp/visualize.jsp]: I changed it to `localGenesetList`....

**Comment by pieterlukasse** [comment]
@sandertan thanks! Can you please ensure this becomes part of https://github.com/cBioPortal/cbioportal/pull/2476 and then close this one?

**Comment by sandertan** [comment]
I added this to #2476.

**Pull request closed without merging**